Assets/MixedRealityToolkit/Inspectors/Utilities/AssemblyExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
```

Comment:
Note that this is probably better located in this folder and in this namespace:

https://github.com/microsoft/MixedRealityToolkit-Unity/tree/mrtk_development/Assets/MixedRealityToolkit/Extensions

Just to be consistent with the rest of the extension.

If this is truly editor-only (I'm not sure about this, but just listing this as an option in case it's relevant), it can go here:

https://github.com/microsoft/MixedRealityToolkit-Unity/tree/mrtk_development/Assets/MixedRealityToolkit/Extensions/EditorClassExtensions

Reply:
Ok, moved it next to the other extension classes and updated the namespace.

Reply:
@wiwei , addressed your comment.
Can you
`/azp run`
for me?